doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -80,11 +80,16 @@ described elsewhere
 
 .. cmd:: Check @term
 
-   Displays the type of :n:`@term`. When called in proof mode, the
-   term is checked in the local context of the selected goal.
+   Displays the type of :n:`@term`. When called in proof mode, the term is
+   checked in the local context of the selected goal. This command tries to
```

Comment:
Probably worth making explicit that single numbered goal selectors can be used to select a goal.